src/explore-education-statistics-common/src/modules/table-tool/components/FiltersForm.tsx

```diff
@@ -115,20 +125,30 @@ const FiltersForm = (props: Props & InjectedWizardProps) => {
   const handleSubmit = async (values: FormValues) => {
     setPreviousValues(values);
     try {
+      setTableQueryError(undefined);
       await onSubmit(values);
-      toggleTableSizeError.off();
       goToNextStep();
     } catch (error) {
       if (
-        isServerValidationError(error, 'QUERY_EXCEEDS_MAX_ALLOWABLE_TABLE_SIZE')
+        isServerValidationError(
+          error,
+          'QUERY_EXCEEDS_MAX_ALLOWABLE_TABLE_SIZE',
```

Comment:
Not super keen on `isServerValidationError` accepting an error message argument. `isServerValidationError` is a type guard meaning that's specifically meant to narrow down the type i.e. it allows us to assert that a value is a `AxiosError<ServerValidationErrorResponse>`.

If we allow it to accept error message arguments and return true/false depending on if these are present inside of it, then this skews the semantics as receiving a `AxiosError<ServerValidationErrorResponse>` should still return true even if the response doesn't contain the specific error we're looking for.

Would suggest that we refactor this so that `isServerValidationError` no longer takes error messages to check against, and instead we use a function like `hasServerValidationError` instead